python/nammu/controller/MenuController.py

```diff
@@ -36,6 +35,9 @@ def saveFile(self):
     def closeFile(self):
         self.controller.closeFile()
     
+    def printFile(self):
```

Comment:
Code refactored with __getattr__ in the ToolbarController and the MenuController to delegate things to the main controller, like so:

```
    #Whenever a MenuController's method is invoked, __getattr__ will search for 
    #that given method name in this class. If it's not found, it'll delegate the
    #action with same name to NammuController
    def __getattr__(self, name):
        return getattr(self.controller, name)
```